packages/react/firebase.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
moving `firebase.json` and `.firebaserc` into `packages/react` and `packages/angular` seems to introduce a breaking change to some scripts at the root workspaces [package.json](https://github.com/invertase/tanstack-query-firebase/blob/main/package.json)

given the contents of these are the same, i'm curious if we want to move them into package dirs

Reply:
Updated